### ISSUE TITLE
stream doesn't support multichannel communication

### ISSUE THREAD

**Issue by tomplus** [tomplus/kubernetes_asyncio#12]
Current implementation of web-socket doesn't support multichannel communication.

Now it's impossible to "talk" to a pod like sending a command, reading output, send another command ... in the same connection.

**Comment by futianshi1314** [comment]
I know that i will close the issues...

**Comment by futianshi1314** [comment]
https://github.com/tomplus/kubernetes_asyncio/issues/12   closed

**Comment by athornton** [comment]
Oof.  This is biting me as I try to get the test suite working for a kubernetes_asyncio-based JupyterHUb Kubespawner.  I'll see if I can figure any of that out.

**Comment by bobh66** [comment]
@tomplus - I'm finishing up the DynamicClient work for #209 hopefully this week, and then I'll probably take a shot at this, or at least dig into it and see how much work it is.

**Comment by bobh66** [comment]
I have a working implementation of stream() - portforward() is a bit more complicated and I'm still working on that.

**Comment by zyairzy** [comment]
In readme, it says "read-only is already implemented", and this issue is still open. But I see [#328 ](https://github.com/tomplus/kubernetes_asyncio/pull/328), it seems like can interactive with pod by stream. It make me confused.
May I ask if the project still does not support bidirectional communication with pods?
I want to use this package in fastapi to interactive with pod, by exec command and show response.

**Comment by tomplus** [comment]
Good point! It's possible to do it as you can see in the example but my intention was to make it more user friendly and hide some low level details.
